rhttpc-client/src/main/scala/rhttpc/client/proxy/ReliableProxy.scala

```diff
@@ -117,7 +117,7 @@ object ExhaustedRetry {
   def apply(cause: Throwable): ExhaustedRetry = ExhaustedRetry(s"Exhausted retry. Message will be moved to DLQ. Cause: ${cause.getMessage}")
 }
 
-case class ReliableProxyFactory()(implicit actorSystem: ActorSystem) {
+case class ReliableProxyFactory(implicit actorSystem: ActorSystem) {
```

Comment:
It will work with scala 2.11? Why you changed it? I don't remember why I've used here case classes (maybe it was bad design choice) but case class with only implicit parameter looks scary...

Reply:
I think I did that accidentally, but its true case class with just implicit seems weird